snorkel/annotations.py

```diff
@@ -61,7 +61,7 @@ def _get_sliced_indexes(self, s, axis, index):
         else:
             idxs = np.array([s])
         index_new, inv_index_new = {}, {}
-        for i, k in index.iteritems():
+        for i, k in index.items():
```

Comment:
I think this loads them all into memory- would like to avoid this.  Unless am wrong here?